ox3apiclient/__init__.py

```diff
@@ -42,13 +39,20 @@
 API_PATH_V2 = '/ox/4.0'
 API_PATH_SSO = '/api'
 ACCEPTABLE_PATHS = (API_PATH_V1, API_PATH_V2, API_PATH_SSO)
-JSON_PATHS = (API_PATH_V2,)
+JSON_PATHS = (API_PATH_V2)
```

Comment:
You are correct @haohany I will bring this branch as it is and send a PR with several fixes that might be missing (like changes on setup.py)